extensions/notebook/src/book/bookVersionHandler.ts

```diff
@@ -56,6 +56,13 @@ export function convertFrom(version: string, section: JupyterBookSection): Jupyt
 	}
 }
 
+function convertNotNumbered(section: JupyterBookSection): boolean | undefined {
```

Comment:
Consider adding a doc comment as well - I still don't think this is very clear what it means just from the name. What is it converting and into what? Ideally someone should be able to tell just from the name alone - or at least from reading the doc comment.